crates/ruff_workspace/src/options.rs

```diff
@@ -3406,12 +3433,20 @@ pub struct AnalyzeOptions {
 
 #[cfg(test)]
 mod tests {
+    use crate::options::Flake8ImportConventionsOptions;
     use crate::options::Flake8SelfOptions;
     use ruff_linter::rules::flake8_self;
     use ruff_linter::settings::types::PythonVersion as LinterPythonVersion;
     use ruff_python_ast::name::Name;
     use ruff_python_formatter::PythonVersion as FormatterPythonVersion;
 
+    #[test]
+    fn flake8_import_conventions_validate_aliases() {
+        let json_options = r#"{"aliases": {"a.b":"a.b"}}"#;
+        let result: Result<Flake8ImportConventionsOptions, _> = serde_json::from_str(json_options);
+        assert!(result.unwrap_err().to_string().contains("Module must be valid identifier separated by single periods and alias must be valid identifier"));
+    }
+
```

Comment:
I'm leaning towards making this a ruff cli integration test. It then also demonstrates how the error message is displayed to the users. 

See https://github.com/astral-sh/ruff/blob/c847cad389f202edae868765e690cb7042e88264/crates/ruff/tests/lint.rs#L23

Reply:
Done!